File: ChickenScratch/chicken-scratch-app/src/app/models/gameState.ts
import { Player } from "./player";

export interface GameState {
    players: GamePlayer[],
    activePlayer: GamePlayer,
    currentWord: string,
    startOfNewRound: boolean
}

export interface GamePlayer {
    id: string;
    name: string;
    score: number;
    hasGuessedCorrectly: boolean;
}

File: ChickenScratch/Models/GameState.cs
using ChickenScratch.Game;
using System.Collections.Generic;

namespace ChickenScratch.Models
{
    public class GameState
    {
        public List<GamePlayer> Players { get; set; }
        public GamePlayer ActivePlayer { get; set; }
        public string CurrentWord { get; set; }
    }
}
